client/src/components/aggregations/CalendarWidget.js

```diff
@@ -6,6 +6,7 @@ import {
   aggregationWidgetDefaultProps,
   aggregationWidgetPropTypes
 } from "components/aggregations/utils"
+import { reportsToEvents } from "components/ReportCalendar"
```

Comment:
Hm, I don't like the fact that this widget, which is itself not report-specific, imports from `ReportCalendar`. I suggest to refactor this.

Reply:
Yes, i was planning to ask this part, we need type of objects information + how to make them events, something like this:
`GET_EVENTS_FUNCTIONS = {
  Report: reportsToEvents,
  Task: tasksToEvents, // will be implemented when needed
 etc...
}
`

i will try to implement this, maybe create another file where all the object => calendarEvent transformations exist

Reply:
These transformations might go into the already existing `client/src/components/aggregations/utils.js` (seems as good a place as any).